systemtest/src/test/java/io/strimzi/systemtest/ConnectST.java

```diff
@@ -478,7 +478,7 @@ void testSecretsWithKafkaConnectWithTlsAndScramShaAuthentication() throws Except
 
         SecretUtils.waitForSecretReady(userName);
 
-        KafkaConnectResource.kafkaConnect(CLUSTER_NAME, 1)
+        KafkaConnectResource.kafkaConnect(CLUSTER_NAME, 1, false)
```

Comment:
In all this deployments of Kafka connect will be really great to have encapsulation of this third parameter for the default value...